.storybook/main.ts

```diff
@@ -10,12 +10,15 @@ const config: StorybookConfig = {
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
     'storybook-dark-mode',
-    '@chakra-ui/storybook-addon',
   ],
   framework: {
     name: '@storybook/react-vite',
     options: {},
   },
+  features: {
+    // @ts-ignore
+    emotionAlias: false,
```

Comment:
Should we document this line as it isn't an official option? what does it resolve?

Reply:
yeah, right
should've documented it

Chakra plugin wants it. But I think we can just drop the chakra plugin, it doesn't do much, because we have a strange setup :D
Not using it on latest version at all